tests/test_validation.py

```diff
@@ -107,3 +107,10 @@ def test_main_geometry(here: Path) -> None:
     assert error.message == (
         "mainGeometry = 'merkOmschrijving' is not a geometry field, type = 'string'"
     )
+
+
+def test_rel_auth(here: Path) -> None:
+    dataset = dataset_schema_from_path(here / "files" / "rel_auth.json")
```

Comment:
This does not test all the scenarios for `auth`. 
Maybe it is easier to read in the json for the dataset into a dict and tweak it on the fly and then use `DatasetSchema.from_dict`, instead of writing out all datasets in json format.

Reply:
Done.